src/Tmds.Ssh/LoggingExtensions.cs

```diff
@@ -180,7 +180,7 @@ public PacketPayload(ReadOnlyPacket packet)
 
             public override string ToString()
             {
-                const int maxDataLength = 2 * PrettyBytePrinter.BytesPerLine;
+                const int maxDataLength = 20 * PrettyBytePrinter.BytesPerLine;
```

Comment:
Should we update this to `20`?

Reply:
No, I guess not. It was just for my personal use, removed